google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionStatementExecutorImpl.java

```diff
@@ -170,6 +181,26 @@ public StatementResult statementShowCommitTimestamp() {
         "COMMIT_TIMESTAMP", getConnection().getCommitTimestampOrNull(), SHOW_COMMIT_TIMESTAMP);
   }
 
+  @Override
+  public StatementResult statementShowCommitResponse() {
+    CommitResponse response = getConnection().getCommitResponseOrNull();
+    CommitStats stats =
+        response == null || !response.hasCommitStats() ? null : response.getCommitStats();
```

Comment:
This is a little too complex to read as a ternary. Consider using if blocks.

Reply:
Done.